packages/test/test-end-to-end-tests/src/test/gc/gcInactiveNodes.spec.ts

```diff
@@ -119,11 +119,7 @@ describeNoCompat("GC inactive nodes tests", (getTestObjectProvider) => {
 
 		itExpects(
 			"can generate events when unreferenced data store is accessed after it's inactive",
-			[
-				{ eventName: changedEvent, timeout: inactiveTimeoutMs },
-				{ eventName: loadedEvent, timeout: inactiveTimeoutMs },
-				{ eventName: revivedEvent, timeout: inactiveTimeoutMs },
-			],
+			[{ eventName: changedEvent }, { eventName: loadedEvent }, { eventName: revivedEvent }],
```

Comment:
Why make this change?

Reply:
I guess timeout is probbly in details now?

Reply:
Yes. We validate the timeout explicitly in the test so this doesn't reduce the validation that was happening.